src/collective/nitf/upgrades/v1008/__init__.py

```diff
@@ -12,7 +12,10 @@ def fix_collections(context):
     logger.info('About to update {0} objects'.format(len(collections)))
     for col in collections:
         obj = col.getObject()
-        query = obj.getQuery()
+        try:
+            query = obj.getRawQuery()
```

Comment:
@hvelarde this method in this place (`getQuery` or `getRawQuery`) is intended to return a list of dicts with the query of the collection.

The problem is that when there is available `getRawQuery`, the `getQuery` method return other class instead of the list of dicts.. it return a list of instances of `CatalogContentListingObject`, what makes the original exception.

This way I check both environment.. when buildout provides the `getQuery` and `getRawQuery` methods and when there is just `getQuery` method available.